Formula/s/seaweedfs.rb

```diff
@@ -7,6 +7,11 @@ class Seaweedfs < Formula
   license "Apache-2.0"
   head "https://github.com/seaweedfs/seaweedfs.git", branch: "master"
 
+  livecheck do
+    url :stable
+    regex(/^(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+)$/i)
```
This would be consistent with other formulae.